### PR TITLE
PAY-663: Payments overview topic

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

This pull request (PR) creates a Payments overview topic for the top-level Magento Payments section documentation.

Preview build: https://docs.magedevteam.com/67/user-guide/magento-payments/payments.html
Jira: https://jira.corp.magento.com/browse/PAY-663

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/payment/payments.html
- 
## Links to Magento source code or PRs

- ...
- ...

### CODE REVIEW

File: src/magento-payments/payments.md [hunk 1]
```diff
@@ -1,3 +1,84 @@
 ---
 title: Magento Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution, including sandbox testing and a super simple setup, for providing robust and secure payment processing for your Magento websites.
+
+Whether you are a small business, mid-market contender, or large enterprise, our white-labeled pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue, driving greater merchant adoption rates, and vastly improving the user's checkout experience.
```
Comment (nofuss) [inline src/magento-payments/payments.md:7]: Is "white-labeled" correct? Isn't it labelled as "Magento Payments"?

File: src/magento-payments/payments.md [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+<!-- ## Architectural overview
+
+See detailed information about the architecture of Magento Payments in the [Architecture topic in DevDocs](https://devdocs.magento.com/magento-payments/architecture.html). -->
+
+## Get started
+
+Onboarding and setting up Magento Payments is a quick process. Setup and configuration is completed in just a few steps:
+
+1. Connect your Magento instance to Magento Software as a Service (SaaS) services.
+1. Onboard and set up the sandbox service.
```
Comment (nofuss) [inline src/magento-payments/payments.md:44]: (quibble) "set up the sandbox service" sounds a bit off for me - maybe something like "onboard to the Magento Payments sandbox service"
the next step could just be "Enable Magento Payment as your payment method and start processing test payments."
(end quibble)

**Review by nofuss — COMMENTED** [review]
not much to complain about

File: src/magento-payments/payments.md (hunk 1 above)
Comment (shrielenee) [reply src/magento-payments/payments.md:7]: This was from previous text/copy. I think you're right. Changing.

File: src/magento-payments/payments.md (hunk 2 above)
Comment (shrielenee) [reply src/magento-payments/payments.md:44]: I totally understand this comment. My only thing is "onboard to the Magento Payments sandbox service" makes sense to us. But I'm not sure the average merchant understands that as well as they would when adding the "set up" part. I'll have to rethink this. Good feedback.

**Review by cherdman — APPROVED** [review]
Looks great, Shrie.

**Review by crmccann — COMMENTED** [review]
Few modifications to suggest:

Whether you are a small business, mid-market contender, or large enterprise, our pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue, driving greater merchant adoption rates, and vastly improving the user’s checkout experience.

--this is the modified text below. It's adobe who gets greater adoption rates, not the merchant--
Whether you are a small business, mid-market contender, or large enterprise, our pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue, giving you useful tools to improve the entire shopper experience from checkout through return and refund.

--an modification here--
Magento Payments is…
Safe and secure (PCI SAQ-A ready)

--an addition here--
Getting started
- get the extension from *Marketplace


Features
Magento Payments is your one-stop-shop for online checkout.
--this is the modified text below.--
Magento Payments is your one-stop-shop for full payments lifecycle from checkout through settle, refund and get paid.

**Comment by shrielenee** [comment]
> Few modifications to suggest:
> 
> Whether you are a small business, mid-market contender, or large enterprise, our pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue, driving greater merchant adoption rates, and vastly improving the user’s checkout experience.
> 
> --this is the modified text below. It's adobe who gets greater adoption rates, not the merchant--
> Whether you are a small business, mid-market contender, or large enterprise, our pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue, giving you useful tools to improve the entire shopper experience from checkout through return and refund.
> 
> --an modification here--
> Magento Payments is…
> Safe and secure (PCI SAQ-A ready)
> 
> --an addition here--
> Getting started
> 
> * get the extension from *Marketplace
> 
> Features
> Magento Payments is your one-stop-shop for online checkout.
> --this is the modified text below.--
> Magento Payments is your one-stop-shop for full payments lifecycle from checkout through settle, refund and get paid.

@crmccann Is the merchant going to know what SAQ-A is? I am reticent to put undefined acronyms in the documentation.

Also, in my Onboarding topic (already created) I did not mention obtaining the extension from Marketplace. I guess I will need to add that to the topic?

File: src/magento-payments/onboarding-payments.md [hunk 3]
```diff
@@ -124,6 +126,4 @@ After you [configure your Magento Services](#configure-magento-services), enable
 
 ## Get help
 
-Contact info
-
-### Log Support ticket
\ No newline at end of file
+The onboarding process is designed to guide you through the required steps for setting up and enabling all Magento Payments functionality. However, you may have questions along the way! See the [Support section of our Payments overview topic]({% link magento-payments/payments.md %}#support) to get the help you need, fast.
```
Comment (leslietilling) [inline src/magento-payments/onboarding-payments.md:129]: I would discourage use of exclamation here.

File: src/magento-payments/payments.md [hunk 4]
```diff
@@ -1,3 +1,58 @@
...
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution, including sandbox testing and a super simple setup, for providing robust and secure payment processing for your Magento websites.
+
+Whether you are a small business, mid-market contender, or large enterprise, our pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue and giving you useful tools to improve the entire shopper experience.
+
+All the tools needed to implement and operate Magento Payments are fully integrated into the Magento Admin---manage everything in one spot with ease.
+
+Magento Payments is...
```
Comment (leslietilling) [inline src/magento-payments/payments.md:11]: no `...` here

File: src/magento-payments/payments.md [hunk 5]
```diff
@@ -1,3 +1,58 @@
...
+* Self-contained within the Magento Admin
+
+IMAGE
+
+## Features
+
+{:.bs-callout-info}
+Some of the features mentioned here may not yet be available for the Early Adopter Program (EAP).
+
+Magento Payments is your one-stop-shop for online checkout (from settlement and refunds to getting paid!) We make setup and configuration painless and we provide powerful tools to give you the insight and control you need to create the best experience for your buyers.
```
Comment (leslietilling) [inline src/magento-payments/payments.md:26]: As a general rule, exclamations are a little over the top in the tech docs. Marketing and tweets can get away with it a bit more.

**Review by leslietilling — APPROVED** [review]
Looking good!  Just a couple of style comments.

File: src/magento-payments/payments.md (hunk 5 above)
Comment (shrielenee) [reply src/magento-payments/payments.md:26]: I understand what you are saying. My hope was that, given our non-technical audience, we could inject a bit more personality into the documentation to instill some confidence in this new product.

**Pull request merged**